common/v2/features/AddAccount/components/SaveAndRedirect.tsx

```diff
@@ -19,7 +24,11 @@ function SaveAndRedirect(payload: { formData: FormData }) {
   const { displayNotification } = useContext(NotificationsContext);
   useEffect(() => {
     const network: Network | undefined = getNetworkByName(payload.formData.network);
-    if (!network || getAccountByAddress(payload.formData.account)) {
+    if (
+      !network ||
+      getAccountByAddressAndNetwork(payload.formData.account, payload.formData.network) !==
```

Comment:
Prefer the double bang (!!) syntax to make sure you receive the account
`!!getAccountByAddressAndNetwork(...)`